src/Microsoft.Framework.Runtime.Interfaces/IAssemblyLoaderContainer.cs

```diff
@@ -6,8 +6,16 @@
 
 namespace Microsoft.Framework.Runtime
 {
+    /// <summary>
+    /// Provides access to assembly loaders used for runtime assembly resolution.
+    /// </summary>
     public interface IAssemblyLoaderContainer
     {
+        /// <summary>
+        /// Adds an <see cref="IAssemblyLoader"/> to the list of loaders.
+        /// </summary>
+        /// <param name="loader">The loader to add.</param>
+        /// <returns>A disposable representing the registration of the loader. Disposing it removes the loader from the list.</returns>
```

Comment:
Yeah "Disposing it removes the loader from the runtime" is probably a little more accurate. Agree with "disposable object" as well, or possibly "disposable token"